tutorial/markdown/nodejs/express.md

```diff
@@ -22,343 +21,335 @@ sdk_language:
+```sh
+docker run -it --env-file config/dev.env -p 3000:3000 couchbase-nodejs-quickstart
+```
 
-We will be setting up a REST API to manage some profile documents. Our profile document will have an auto-generated UUID for its key, first and last name of the user, an email, and hashed password. For this demo we will store all profile information in just one document in a collection named `profile`:
+> Note: The `config/dev.env` file has the connection information to connect to your Capella cluster. With the `--env-file`, docker will inject those environment variables to the container.
 
-```json
-{
-  "email": "johnwick@couchbase.com",
-  "password": "$2a$10$tZ23pbQ1sCX4BknkDIN6NekNo1p/Xo.Vfsttm.USwWYbLAAspeWsC",
-  "firstName": "John",
-  "lastName": "Wick",
-  "pid": "b181551f-071a-4539-96a5-8a3fe8717faf"
-}
-```
+Once the app is up and running, you can launch your browser and go to the [Swagger documentation](https://localhost:3000/) to test the APIs.
 
-## Let's Review the Code
+### Verifying the Application
 
-We will be going over the `App.js` file located in the `/src` directory. Let's clone the repo onto our machine, get it up and running, and open the `App.js` to learn about how to create, read, update and delete documents in our Couchbase Server.
+Once the application starts, you can see the details of the application on the logs.
 
-**`src/server.js`:**
+![Application Startup](nodejs_app_startup.png)
 
-The `server.js` file bootstraps our application by importing our `app.js` file and runs the `ensureProfileIndex()` before starting up the application, as we need those indexes for any use of N1QL queries in our application:
+The application will run on port 3000 of your local machine (http://localhost:3000). You will find the interactive Swagger documentation of the API if you go to the URL in your browser. Swagger documentation is used in this demo to showcase the different API end points and how they can be invoked. More details on the Swagger documentation can be found in the [appendix](#swagger-documentation).
 
-```js
-import { app, ensureProfileIndex } from './app.js'
-
-const startApiServer = async() => {
-  await ensureProfileIndex()
-    .then(() => {
-      app.listen(process.env.APP_PORT,
-        () => console.info(`Running on port ${process.env.APP_PORT}...`)
-      )
-    })
-}
+![Swagger Documentation](nodejs_swagger_documentation.png)
 
-startApiServer()
-```
+## Data Model
 
-**`src/app.js`:**
+For this tutorial, we use three collections, `airport`, `airline` and `route` that contain sample airports, airlines and airline routes respectively. The route collection connects the airports and airlines as seen in the figure below. We use these connections in the quickstart to generate airports that are directly connected and airlines connecting to a destination airport. Note that these are just examples to highlight how you can use SQL++ queries to join the collections.
+![img](travel_sample_data_model.png)
```

Comment:
You need to attach the image. It is not rendering currently.

Reply:
Attached missing image

Reply:
Can you add a newline before the image to avoid wrapping the text around the image?